control-plane/main.tf

```diff
@@ -190,16 +191,16 @@ module "install_common" {
   rancher_version     = var.rancher_version
   certmanager_version = var.certmanager_version
 
-  helm_rancher_chart_values_path = "${path.module}/files/values/rancher_chart_values.tftpl"
-  letsencrypt_email              = var.letsencrypt_email
-  rancher_image                  = var.rancher_image
-  rancher_image_tag              = var.rancher_image_tag
-  rancher_password               = var.rancher_password
-  use_new_bootstrap              = local.use_new_bootstrap
-  rancher_node_count             = var.rancher_node_count
-  byo_certs_bucket_path          = var.byo_certs_bucket_path
-  private_ca_file                = var.private_ca_file
-  cattle_prometheus_metrics      = var.cattle_prometheus_metrics
+  # helm_rancher_chart_values_path = "${path.module}/files/values/rancher_chart_values.tftpl"
```

Comment:
Why is this commented out?

Reply:
This is now handled within the `install_common` module itself, so we are using the default `helm_rancher_chart_values_path` that is supplied by the module

Reply:
We are using the default `helm_rancher_chart_values_path` which is supplied by the `install_common` module now. But this comment can probably be deleted

Reply:
No worries, just curious.